### DISCUSSION TITLE
como configurar atraves de media queries a versão desktop no celular

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#1759] [Perguntas e Respostas]
Boa tarde ,

eu sou mais focado em interfaces design mesmo , mais estou fazendo um projero no qual fiz o front end, e surgiu uma dúvida , quando o usuário esta no celular e clica no item versão para computar , existe uma forma de configurar via media queries

**Comment by portothree** [comment]
Se estiver se referindo ao "Ver em modo desktop" presente no menu de alguns navegadores mobile, o que ele faz é ignorar o width e scaling na meta tag do html.
Um exemplo de meta tag para viewport:
```html
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```
